### DISCUSSION TITLE
How can I enable hot reloading of frontend changes

### DISCUSSION THREAD

**Discussion by ralfgehrer** [fastapi/full-stack-fastapi-template#1195] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
-
```

### Description

- run docker-compose up -d --build
- open the web app and login with the admin user
- E.g. add a menu item in UserMenu.tsx --> I'd like to see the changes immediately (after refreshing the browser)
- But instead I have to down and up the docker compose to see my changes

### Operating System

macOS

### Operating System Details

Intel Processor 

### Python Version

latest - see docker compose

**Comment by alejsdev** [comment, marked as answer]
Hi @ralfgehrer, building the image at every change will not be as productive as running the local development server with live reload. It's documented in the same docs, you can check it here: https://github.com/tiangolo/full-stack-fastapi-template/tree/master/frontend#frontend-development :rocket:
